docs/kit/logistics/kit-loan-extension.md

```diff
@@ -0,0 +1,27 @@
+# Kit Loan Extension
+
+As per appendix A.2 of the [rules][rules], teams may request to keep the kit after the competition.
```

Comment:
Since this is a link to the generic rules page, and therefore the specific appendix may change, maybe be less specific?

```suggestion
In general we expect the teams to return all their kits at the Competition. However we know that some teams prefer to keep their kits for a few weeks afterwards in order, for example, to show off the robots at school. In order to facilitate this, teams may request a kit loan extension.

We specify the parts which the teams must return, as well as the mechanism and deadline for loan extension requests, within an appendix to the [rules][rules].
```